Makefile

```diff
@@ -22,4 +23,7 @@ clean-python:
 
 .PHONY: package-node
 package-node:
-	./scripts/package-node.sh
+	@./scripts/package-node.sh
+
+.PHONY: packages
```

Comment:
👌 Neat!

Reply:
Happy to rename this one if you have a better name, this was just a placeholder for it for now 😄